src/Compilers/Core/Portable/MetadataReference/Metadata.cs

```diff
@@ -1,7 +1,6 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
 using System;
-using Microsoft.CodeAnalysis.Text;
```

Comment:
I see, it's a delete :)

---
In reply to: [62749485](https://github.com/dotnet/roslyn/pull/11183#discussion_r62749485) [](ancestors = 62749485)